repository-data/webfiles/src/main/resources/site/freemarker/intranet/macro/trendingArticlesDate.ftl

```diff
@@ -0,0 +1,18 @@
+<#ftl output_format="HTML">
+
+<#macro trendingArticleDate trending>
+    <#if trending.docType == 'NewsInternal'>
+        <#if trending.publicationDate??>
+            <span class="intra-home-article-grid-article__date">
+                 <@fmt.formatDate value=trending.publicationDate.time type="Date" pattern="d MMMM yyyy" timeZone="${getTimeZone()}"/>
+            </span>
+        </#if>
+    <#elseif trending.docType == 'Blog'>
+        <#if trending.dateOfPublication??>
+            <span class="intra-home-article-grid-article__date">
+                <@fmt.formatDate value=trending.dateOfPublication.time type="Date" pattern="d MMMM yyyy" timeZone="${getTimeZone()}"/>
+            </span>
+        </#if>
+    </#if>
```

Comment:
Can this else block be combined with the ` <#if trending.docType == 'NewsInternal'>` block above? Or is there a reason not to?

Reply:
No, the date fields are different on the two document types: 'publicationDate' & 'dateOfPublication'